java/google/registry/request/BUILD

```diff
@@ -21,15 +19,15 @@ java_library(
         "@com_googlecode_json_simple",
         "@javax_servlet_api",
         "@joda_time",
+        "//java/google/registry/security",
```

Comment:
Hey Ben, the reordering is being done by the Intellij Bazel plugin. I've not found a way to modify or disable to import ordering with that plugin outside of completely deleting it. If this is an incorrect order then there is most likely a bug with that plugin.